cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt

```diff
@@ -310,6 +461,69 @@ policy "pol_turbonomic_rightsize_volumes" do
       field "createdTime" do
         label "Created Time"
       end
+      field "attachedVm" do
+        label "Attached VM"
+      end
+      field "actionState" do
+        label "Action State"
+      end
+      field "disruptiveness" do
+        label "Disruptiveness"
+      end
+      field "reversibility" do
+        label "Reversibility"
+      end
+      field "iops" do
+        label "Current IOPS Capacity"
+        path "currentIopsCapacity"
+      end
+      field "newIops" do
+        label "After IOPS Capacity"
+        path "afterIopsCapacity"
+      end
+      field "iopsUtilization" do
+        label "Current IOPS Utilization (%)"
+        path "currentIopsUtilization"
+      end
+      field "newIopsUtilization" do
+        label "After IOPS Utilization (%)"
+        path "afterIopsUtilization"
+      end
+      field "size" do
+        label "Current Size Capacity (GB)"
+        path "currentSizeCapacity"
+      end
+      field "newSize" do
+        label "After Size Capacity (GB)"
+        path "afterSizeCapacity"
+      end
+      field "sizeUtilization" do
+        label "Current Size Utilization (%)"
```

Comment:
"Current" word in all labels need to be removed, 
Change "Size" by "Storage" in all labels.
E.g. "Storage % Utilization"